cdap-ui/app/cdap/styles/fonts/symbol-defs.svg

```diff
@@ -3143,11 +3143,11 @@
 </symbol>
 <symbol id="icon-lock_close" viewBox="0 0 32 32">
 <title>lock_close</title>
-<path d="M24.7 28.4v-14.6h-17.4v14.6h17.4zM11.5 8v2.9h9v-2.9c0-1.2-0.4-2.3-1.3-3.2s-1.9-1.3-3.2-1.3-2.3 0.4-3.2 1.3c-0.9 1-1.3 2-1.3 3.2zM24.7 11c0.8 0 1.5 0.3 2 0.9 0.6 0.6 0.9 1.2 0.9 2v14.6c0 0.8-0.3 1.4-0.9 2s-1.3 0.9-2 0.9h-17.4c-0.8 0-1.5-0.3-2-0.9-0.6-0.6-0.9-1.2-0.9-2v-14.6c0-0.8 0.3-1.4 0.9-2s1.3-0.9 2-0.9h1.4v-3c0-2 0.7-3.7 2.1-5.1s3.1-2.1 5.1-2.1c2 0 3.7 0.7 5.1 2.1s2.1 3.1 2.1 5.1v2.9c0.2 0.1 1.6 0.1 1.6 0.1z"></path>
+<path d="M26.2 13h-0.6v-2.8c0-5.2-4.1-9.6-9.3-9.7-0.1 0-0.4 0-0.6 0-5.2 0.1-9.3 4.5-9.3 9.7v2.8h-0.6c-0.9 0-1.7 1-1.7 2.1v14.2c0 1.2 0.8 2.2 1.7 2.2h20.4c0.9 0 1.7-1 1.7-2.2v-14.1c0-1.2-0.7-2.2-1.7-2.2v0zM17.9 22.2v4.3c0 0.5-0.4 0.9-0.9 0.9h-2c-0.5 0-0.9-0.4-0.9-0.9v-4.3c-0.5-0.5-0.8-1.1-0.8-1.8 0-1.4 1.1-2.5 2.4-2.6 0.1 0 0.4 0 0.6 0 1.3 0.1 2.4 1.2 2.4 2.6 0 0.7-0.3 1.4-0.8 1.8v0zM21.6 13h-11.2v-2.8c0-3.1 2.5-5.7 5.6-5.7s5.6 2.6 5.6 5.7v2.8z"></path>
 </symbol>
 <symbol id="icon-lock_open" viewBox="0 0 32 32">
 <title>lock_open</title>
-<path d="M26.7 11.8c-0.6-0.6-1.3-0.8-2-0.8h-1.4v-3c0-2-0.7-3.7-2.1-5.1s-3.1-2.1-5.1-2.1-3.7 0.7-5.1 2.1c-0.3 0.3-0.6 0.7-0.9 1.1-0.5 0.7-0.3 1.7 0.5 2.1v0c0.6 0.3 1.4 0.1 1.7-0.5 0.2-0.2 0.4-0.5 0.6-0.7 0.9-0.9 1.9-1.3 3.2-1.3s2.3 0.4 3.2 1.3c0.9 0.9 1.3 1.9 1.3 3.2v2.9h-13.2c-0.8 0-1.5 0.3-2 0.8s-0.9 1.2-0.9 2v14.5c0 0.8 0.3 1.4 0.9 2s1.3 0.8 2 0.8h17.4c0.8 0 1.5-0.3 2-0.8 0.6-0.6 0.9-1.2 0.9-2v-14.5c-0.1-0.7-0.4-1.4-1-2zM24.7 28.4h-17.4v-14.5h17.4v14.5z"></path>
+<path d="M5.8 13c-0.9 0-1.7 1-1.7 2.1v14.2c0 1.2 0.8 2.2 1.7 2.2h20.4c0.9 0 1.7-1 1.7-2.2v-14.1c0-1.2-0.8-2.1-1.7-2.1h-0.6v-2.8c0-5.2-4.1-9.6-9.3-9.7-0.1 0-0.4 0-0.6 0-2.4 0-4.5 1-6.1 2.5-0.9 0.8-0.8 2.2 0.1 3v0c0.7 0.6 1.9 0.7 2.6 0 1.2-1.1 2.8-1.6 4.6-1.4 2.8 0.4 4.8 3 4.8 5.8v2.6h-15.9zM17.9 22.2v4.3c0 0.5-0.4 0.9-0.9 0.9h-2c-0.5 0-0.9-0.4-0.9-0.9v-4.3c-0.5-0.5-0.8-1.1-0.8-1.8 0-1.4 1.1-2.5 2.4-2.6 0.1 0 0.4 0 0.6 0 1.3 0.1 2.4 1.2 2.4 2.6 0 0.7-0.3 1.4-0.8 1.8z"></path>
```

Comment:
Verified the lock icon. 👍 
<img width="922" alt="screen shot 2017-06-01 at 11 38 04 pm" src="https://cloud.githubusercontent.com/assets/1452845/26713833/6b930f94-4723-11e7-951c-a25befd109ca.png">